Lottie-Legacy.xcodeproj/xcuserdata/bweiss.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this? is has bweiss in its name so I guess it is something local, remove